src/docs/event.yaml

```diff
@@ -0,0 +1,168 @@
+tags:
+    - name: Event
+      description: 'Operations about event'
+paths:
+    /api/event/:id/join:
+    post:
+        tags:
+            - Event
+        summary: Join user to an event
+        description: add user to an event as a confirmed Volunteer
+        operationId: Join user
+        consumes:
+            - application/json
+        produces:
+            - application/json
+        security:
+            - auth_cookie: []
```

Comment:
You are using an auth_cookie without define it first. This is also fixed in the gist.